reducertodo/src/reducers/todo.js

```diff
@@ -0,0 +1,39 @@
+export const initialState = {
+  todos: [{ item: "Learn about reducers", completed: false, id: 3892987589 }]
+};
+
+export function reducer(state, action) {
+  switch (action.type) {
+    case "CHANGE_TODO":
+      return {
+        ...state,
+        item: action.payload
+      };
+
+    case "DELETE_TODO":
+      return {
+        ...state,
+        todos: state.todos.filter(todo => !todo.completed)
+      };
+    case "ADD_TODO":
+      return {
+        todos: [
+          ...state.todos,
+          { item: action.payload, completed: false, id: Date.now() }
+        ]
+      };
+
+    case "TOGGLE_TODO":
+      return {
+        ...state,
```

Comment:
for this section, you don't actually need to spread out state, as you only have 1 property and that property is returning from a `.map` which is a reducer function